### DISCUSSION TITLE
第5节课笔记(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#407] [笔记]
第5节课笔记(24班)

**Comment by yansuoyuli** [comment]
https://blog.csdn.net/m0_58431543/article/details/135683233

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day5.md

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/fde4d382-c4d5-485c-bccd-fbe964d20256

**Comment by flh1107** [comment]
https://www.bilibili.com/read/cv29928398/

**Comment by Xinyi-Cheng** [comment]
https://zhuanlan.zhihu.com/p/679360164

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135771342?spm=1001.2014.3001.5501

**Comment by lindsey-chang** [comment]
https://blog.csdn.net/qq_37397652/article/details/135778297

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135834479?spm=1001.2014.3001.5501

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135791687?spm=1001.2014.3001.5501
